web_generator/lib/src/generate_bindings.dart

```diff
@@ -86,3 +89,20 @@ Future<TranslationResult> generateBindings(
   translator.addInterfacesAndNamespaces();
   return translator.translate();
 }
+
+Future<TranslationResult> generateBindingsForFiles(
+    Map<String, String> fileContents, String output, bool single) async {
+  final cssStyleDeclarations = await _generateCSSStyleDeclarations();
```

Comment:
nit: I think this is okay because they're necessary dependencies for some IDL files, but it naturally does feel weird to do this for when inputs are specified. I'd just add a comment saying that we need to generate the style declarations and element tag map in case they're used in the input files.